src/configLoader.js

```diff
@@ -13,12 +14,18 @@ export function configLoader(configPath) {
         return process.exit(1);
     }
 
+    const rawConfig = fs.readFileSync(configPath, 'utf8');
+    const compiledConfig = rawConfig.replace(/\$\$\$_([A-Za-z]+)_\$\$\$/g, (match, variableName) => {
+        const envVariableName = changeCase.constantCase(variableName);
```

Comment:
Environment variables are case sensitive in bash and node – is there a reason kongfig shouldn't treat them that way as well?

Reply:
not really, i think that's  good idea. i got here for convoluted reasons. the only thing that i would point out here is that it is case-sensitive with respect to the environment variable, only not to the kongfig variable, and i think that we could hypothetically enforce a convention like camel-case on templated vars if we wanted to.

Reply:
not really, i think that's  good idea. i got here for convoluted reasons. the only thing that i would point out here is that it is case-sensitive with respect to the environment variable, only not to the kongfig variable, and i think that we could hypothetically enforce a convention like camel-case on templated vars if we wanted to.